scripts/zones/Valkurm_Dunes/IDs.lua

```diff
@@ -39,7 +39,24 @@ zones[xi.zone.VALKURM_DUNES] =
         AN_EMPTY_LIGHT_SWIRLS          = 7767,  -- An empty light swirls about the cave, eating away at the surroundings...
         GARRISON_BASE                  = 7769,  -- Hm? What is this? %? How do I know this is not some [San d'Orian/Bastokan/Windurstian] trick?
         TIME_ELAPSED                   = 7816,  -- Time elapsed: <number> [hour/hours] (Vana'diel time) <number> [minute/minutes] and <number> [second/seconds] (Earth time)
+        PICK_UP_THE_ITEM               = 7837,  -- You pick up the <item>.
+        TOO_FAR_AWAY                   = 7838,  -- You are too far away.
+        TOO_MANY_IN_PARTY              = 7840,  -- Nothing happens. Your party exceeds the maximum number of <number> members.
+        ALLIANCE_NOT_ALLOWED           = 7841,  -- Nothing happens. You must dissolve your alliance.
+        NOTHING_TRY_AGAIN              = 7842,  -- Nothing happens.  Please wait a moment and try again.
         MONSTERS_KILLED_ADVENTURERS    = 7843,  -- Long ago, monsters killed many adventurers and merchants just off the coast here. If you find any vestige of the victims and return it to the sea, perhaps it would appease the spirits of the dead.
+        RETURN_TO_SEA                  = 7844,  -- You return the <item> to the sea.
+        WHAT_COULD_THAT_BE             = 7845,  -- Wh-whatever could that be!?
+        ITS_COMING_THIS_WAY            = 7846,  -- It's coming this way!  Get prop-prim-prepared! I've got nothing to use against this beasty-weast!
+        SEE_SHINY_SHIMMERING_POINT     = 7847,  -- Hurry! See that shiny-shimmering point? Fish there! Get the treasure-weasure!
+        RIGHT_OVER_THERE_POINT         = 7848,  -- Right over there! The ship-shape-shimmery point!
+        HURRY_UP                       = 7849,  -- Ahhh! Hurry up, hurry up!!!
+        ITS_COMING_FOR_US              = 7850,  -- It's coming! It's coming for ussssss!
+        THREE_OF_THEM                  = 7851,  -- It can'taru be...!  Three of them!!!?
+        NOOOOOO                        = 7852,  -- Noooooo!
+        CRY_OF_ANGUISH                 = 7853,  -- ??? disappears in a cry of anguish.
```

Comment:
From 7843 to 7853, the text IDs look sequential, as if they were used by the same NPC, in succession and exact order.

I guess my question is, do we really need them all defined? Couldn't you just use `7843 + X`?